prow/scripts/cluster-integration/helpers/create-network-with-subnet.sh

```diff
@@ -22,4 +23,8 @@ fi
 
 gcloud compute networks create "${GCLOUD_NETWORK_NAME}" \
  --project="${GCLOUD_PROJECT_NAME}" \
- --subnet-mode=custom
\ No newline at end of file
+ --subnet-mode=custom
+
+gcloud compute networks subnets create "${GCLOUD_SUBNET_NAME}" \
+ --network="${GCLOUD_NETWORK_NAME}" \
+ --range=10.0.0.0/24
```

Comment:
Please add `--region` for consistency and safety

Reply:
done